pkg/odo/cli/component/create.go

```diff
@@ -345,7 +346,11 @@ func (co *CreateOptions) Run() (err error) {
 
 // NewCmdCreate implements the create odo command
 func NewCmdCreate(name, fullName string) *cobra.Command {
-	co := NewCreateOptions()
+	prefClient, err := preference.NewClient()
+	if err != nil {
+		panic("unable to set preference, something is wrong with odo, kindly raise an issue at https://github.com/redhat-developer/odo/issues/new?template=Bug.md")
```

Comment:
I think it is better to use LogErrorAndExit here, instead of panicking. Same comment for other panics.

Reply:
You're right, thanks